tests/playbooks/tests_auto_gateway.yml

```diff
@@ -91,16 +91,16 @@
             type: ethernet
             state: up
             ip:
-              auto_gateway: no
-              dhcp4: no
-              auto6: no
+              auto_gateway: false
+              dhcp4: false
+              auto6: false
```

Comment:
IMHO it would be nicer to have individual commits for different type of changes, then you can move forward with those that are straight-forward like always using `true`/`false` for booleans. Put this in a separate PR and it can be merged IMHO.